frontend/Caddyfile

```diff
@@ -1,12 +1,11 @@
 {
```

Comment:
?
did you need a special setup ? Maybe we can add this to .gitignore

Reply:
Reverted. It should not have been commited